inspire_schemas/records/seminars.yml

```
@@ -32,6 +32,9 @@ properties:
     address:
         $ref: elements/address.json
         title: Location of the seminar
+    captions:
```

we use adjectives for this kind of stuff if possible:
```suggestion
    captioned:
```